gradle/coverage.gradle

```diff
@@ -1,5 +1,3 @@
-apply plugin: 'cobertura'
-apply plugin: 'coveralls'
 cobertura.coverageFormats = ['html', 'xml'] // coveralls plugin depends on xml format report
 //cobertura.coverageIgnoreTrivial = true
-coveralls.sourceDirs = ['src']  //workaround for https://github.com/stevesaliman/gradle-cobertura-plugin/issues/53
\ No newline at end of file
+coveralls.sourceDirs = ['src']  //workaround for https://github.com/stevesaliman/gradle-cobertura-plugin/issues/53
```

Comment:
Nope, I have removed it in my fork. But given coveralls issues, I'm investigating to switch to jacoco and codecov